tests/ui/test_home.py

```diff
@@ -3,21 +3,74 @@
 from pages.desktop.home import Home
 
 
-@pytest.mark.django_db
 @pytest.mark.nondestructive
 def test_there_are_ten_most_popular_extensions(
         my_base_url, selenium, initial_data):
     """Ten most popular add-ons are listed"""
+
     page = Home(selenium, my_base_url).open()
     assert len(page.most_popular.extensions) == 10
 
 
-@pytest.mark.django_db
 @pytest.mark.nondestructive
 def test_most_popular_extensions_are_sorted_by_users(
         my_base_url, selenium, initial_data):
     """Most popular add-ons are sorted by popularity"""
+
     page = Home(selenium, my_base_url).open()
     extensions = page.most_popular.extensions
     sorted_by_users = sorted(extensions, key=lambda e: e.users, reverse=True)
     assert sorted_by_users == extensions
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_that_clicking_on_addon_name_loads_details_page(
+        my_base_url, selenium, addon):
+    """Details page addon name matches clicked addon"""
+
+    page = Home(selenium, my_base_url).open()
+    name = page.featured_extensions.extensions[0].name
+    extension = page.featured_extensions.extensions[0].see_all()
```

Comment:
Yeah, `extension_page`, `themes_page`, etc. Usually if there's only one page object used in a test, I just call it `page` because it's usually pretty clear what page it represents.